.gitignore

```diff
@@ -6,3 +6,4 @@ repo/*
 npm-debug.log
 yarn-error.log
 !.gitkeep
+reactjs.org-translation.iml
```

Comment:
What's this line of code?

Reply:
Removed, IntelliJ added it's config to project root instead `.idea` default config dir.